yarn.lock

```diff
@@ -5911,7 +5905,30 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/express@npm:*, @types/express@npm:^4.17.23":
```

Comment:
This seems like the issue: https://github.com/expressjs/multer/blob/e259a7ee2141f5c89fd3d3e7d6825e706754340e/package.json from https://github.com/expressjs/multer/releases/tag/v2.0.1. We should consider pinning multer? I did this accidentally as I was trying to avoid a nasty merge conflict in `yarn.lock`, so I just deleted and re-created the file.